### PR TITLE
Add metadatafile

### PR DESCRIPTION
**Pull request by VivekVinushanth**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: .connector-store/meta.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "name": "OPA",
+    "owner": "WSO2",
+    "category": "Adaptive Authentication Function",
+    "documentationUrl": "https://github.com/wso2-extensions/identity-application-authz-opa/tree/v1.0.1/docs",
+    "description": "\"invokeOPA\" in adaptive scripts to be able to pass any JSON object as the payload, and from OPA it is possible to process the data object and extract the required information. Then OPA engine will the final authentication or authorization decision will be made by OPA according to the policies that have defined.",
+    "status": "Active",
+    "labels": [],
+    "releases": [
+        {
+            "tagName": "1.0.1",
```
Comment (Avarjana) [inline .connector-store/meta.json]: Should be changes to version 1.0.0 -> 5.10.0

File: .connector-store/meta.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "OPA",
+    "owner": "WSO2",
+    "category": "Adaptive Authentication Function",
+    "documentationUrl": "https://github.com/wso2-extensions/identity-application-authz-opa/tree/v1.0.1/docs",
+    "description": "\"invokeOPA\" in adaptive scripts to be able to pass any JSON object as the payload, and from OPA it is possible to process the data object and extract the required information. Then OPA engine will the final authentication or authorization decision will be made by OPA according to the policies that have defined.",
+    "status": "Active",
+    "labels": [],
```
Comment (Avarjana) [inline .connector-store/meta.json:8]: Don't we have labels for this?

File: .connector-store/meta.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "OPA",
+    "owner": "WSO2",
+    "category": "Adaptive Authentication Function",
+    "documentationUrl": "https://github.com/wso2-extensions/identity-application-authz-opa/tree/v1.0.1/docs",
+    "description": "\"invokeOPA\" in adaptive scripts to be able to pass any JSON object as the payload, and from OPA it is possible to process the data object and extract the required information. Then OPA engine will the final authentication or authorization decision will be made by OPA according to the policies that have defined.",
```
Comment (Avarjana) [inline .connector-store/meta.json]:
```suggestion
    "description": "\"invokeOPA\" in adaptive scripts to be able to pass any JSON object as the payload, and from OPA it is possible to process the data object and extract the required information. Then OPA engine will do the final authentication or authorization decision according to the policies that have defined.",
```

File: .connector-store/meta.json (hunk 2 above)
Comment (VivekVinushanth) [reply .connector-store/meta.json:8]: Nope.

**Review by Avarjana — APPROVED** [review]

**Pull request merged**